### DISCUSSION TITLE
Project name change

### DISCUSSION THREAD

**Discussion by buchdag** [nginx-proxy/acme-companion#798] [Announcements]
Following a discussion with the members of the nginx-proxy org (more info here: https://github.com/nginx-proxy/nginx-proxy/issues/1304) we've agreed to rename this project, with the most probable choice being `nginx-proxy/acme-companion`.

My rationale for the renaming is:
- Docker aren't fond of project using "docker" in their name (that's why the image actual name is `letsencrypt-nginx-proxy-companion` while the GitHub repository name is `docker-letsencrypt-nginx-proxy-companion`), so we should as well remove it.
- now that the project is under the nginx-proxy org, that fact that it is a companion for `nginx-proxy/nginx-proxy` is kinda implied, so we can also remove it.
- as the project has evolved to support more ACME CAs than just Let's Encrypt (namely BuyPass and ZeroSSL), I feel like `acme-companion` would more accurately reflect the goal of the project and does not tie it to one specific provider's commercial name.

The would be combined with the image being pushed to and available from a new `nginxproxy/acme-companion` DockerHub registry (in addition to `jrcs/letsencrypt-nginx-proxy-companion`).

Comments are welcome.

**Comment by callmemagnus** [comment]
I think it's a good idea. The new proposed name is simpler.

**Comment by 73** [comment]
Great choice! Thanks for all your work!

**Comment by jrd** [comment]
I second the idea.
Also if the docker hub image could be rebuild more often/automatically to fix common CVE/security issues. This container is very essential so it needs to be really up to date :-)

**Comment by adamkdean** [comment]
I think this is a good idea!

**Comment by buchdag** [comment]
@jrd what would be a sensible rebuild schedule to you ?

**Comment by buchdag** [comment]
Project name has been changed, docs update will follow.

**Comment by jrd** [comment]
@buchdag well about once or twice per month seems a good enough schedule to maintain security.

**Comment by buchdag** [comment]
@jrd rebuild and push of the default branch's last commit have been scheduled every Monday at 00:00 UTC (d56d1aa9fd1d467479824d0db8b2ab043b469376)
